lib/location.js

```diff
@@ -343,7 +522,27 @@ var LocationState = State.extend(function (self, name, opts) {
     self.display = function() {
         return self.metadata.display_text || self.question_text;
     };
-    
+
+    self.get_viewbox = function() {
+        if (self.country_code) {
```

Comment:
Take this logic out and make it so bounding_box is set at configuration time from the country_boxes values if that is the user pref. Removes the confusion as to precedence.